doc/onboarding/view/step2.md

```diff
@@ -0,0 +1,82 @@
+
+
+## Step 2/5 : Share anonymously data
+
+### URI
+
+`:userID/share`
```

Comment:
It's not only about sharing here, isn't it? It could be better ```agreements``` but I think ```share_agreement``` is not enough explicit here.